This patch was taken originally from OE:
http://cgit.openembedded.org/openembedded/tree/recipes/openais/files/fix-lcrso-linkage.patch

It was updated for OpenAIS v1.1.4

Upstream Status: Pending

Signed-of-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/services/Makefile.am b/services/Makefile.am
index 1e6e62f..b23c8c6 100644
--- a/services/Makefile.am
+++ b/services/Makefile.am
@@ -78,13 +78,13 @@ service_%.lcrso: %.o
 
 else
 service_amf.lcrso: $(AMF_OBJECTS)
-	$(CC) $(CFLAGS) -shared -Wl,-soname=$@ $^ -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared -Wl,-soname=$@ $^ -o $@
 
 service_%.lcrso: %.o
-	$(CC) $(CFLAGS) -shared -Wl,-soname=$@ $^ -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared -Wl,-soname=$@ $^ -o $@
 
 %.lcrso: %.o
-	$(CC) $(CFLAGS) -shared -Wl,-soname=$@ $^ -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared -Wl,-soname=$@ $^ -o $@
 endif
 
 %.o: %.c
